sbt-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -114,14 +147,16 @@ object ScalaNativePluginInternal {
     val flags   = Seq("-o", outpath) ++ linkopts ++ opts
     val compile = abs(clangpp) +: (flags ++ paths)
 
-    Process(compile, target).!
+    logger.info(running(compile))
+
+    Process(compile, target) ! logger
   }
 
   lazy val projectSettings = Seq(
     libraryDependencies ++= Seq(
-      "org.scala-native" %% "nativelib" % nativeVersion,
-      "org.scala-native" %% "javalib"   % nativeVersion,
-      "org.scala-native" %% "scalalib"  % nativeVersion
+      "org.scala-native" %%% "nativelib" % nativeVersion,
+      "org.scala-native" %%% "javalib"   % nativeVersion,
+      "org.scala-native" %%% "scalalib"  % nativeVersion
```

Comment:
nope, `%%%` means to map the artifact name to the current target platform.
before:  nativelib_native_2.11
now: nativelib_native0.1-SNAPSHOT_2.11